pkgs/applications/misc/tiramisu/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, pkg-config, glib }:
+
+stdenv.mkDerivation rec {
+  pname = "tiramisu";
+  version = "1.0";
+
+  src = fetchFromGitHub {
+    owner = "Sweets";
+    repo = pname;
+    rev = version;
+    sha256 = "0aw17riwgrhsmcndzh7sw2zw8xvn3d203c2gcrqi9nk5pa7fwp9m";
+  };
+
+  postPatch = ''
+    sed -i 's/printf(element_delimiter)/printf("%s", element_delimiter)/' src/output.c
+  '';
+
+  buildInputs = [ pkg-config glib ];
```

Comment:
pkg-config is a build tool so it likely goes to `nativeBuildInputs`, not `buildInputs`.

Reply:
Looks like it. For that matter, the [wiki](https://nixos.wiki/wiki/Packaging/Quirks_and_Caveats) mentions it.